integration/test/test_gpu_activity_agent.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2015 - 2022, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+"""
+This integration test verifies that the gpu_activity agent can improve
+efficiency of an application.
+"""
+import sys
+import unittest
+import os
+from pathlib import Path
+import shutil
+from experiment import machine
+from types import SimpleNamespace
+
+import geopmpy.agent
+import geopmpy.io
+
+from integration.test import util
+from integration.test import geopm_test_launcher
+from experiment.energy_efficiency import gpu_activity
+from apps.parres import parres
+
+@util.skip_unless_config_enable('beta')
+@util.skip_unless_do_launch()
+@util.skip_unless_gpu()
+@util.skip_unless_workload_exists("apps/parres/Kernels/Cxx11/dgemm-mpi-cublas")
+@util.skip_unless_workload_exists("apps/parres/Kernels/Cxx11/nstream-mpi-cuda")
```

Comment:
If we remove these the test will fail on run instead of skipping.  This might be a better default behavior if we only plan to run this script when the workloads have been built.

Reply:
Note from Brad: remove. Tests should fall over if apps not built.